helm/defectdojo/values.yaml

```diff
@@ -46,6 +46,9 @@ database: postgresql
 # Primary hostname of instance
 host: defectdojo.default.minikube.local
 
+# The full URL to your defectdojo instance, depends on the domain where DD is deployed, it also affects links in Jira
+site_url: 'http://localhost:8080'
```

Comment:
Would be good to comment out this. You already have defaults in the configMaps.

Reply:
Done !